extra/3RD_PARTY_QUICK_START.md

```diff
@@ -7,24 +7,177 @@ So you'd like to create and share your own language for Highlight.js.  That's aw
 Take a look at some of the real-life examples first:
 
 - https://github.com/highlightjs/highlightjs-cypher
+- https://github.com/highlightjs/highlightjs-structured-text
 - https://github.com/highlightjs/highlightjs-robots-txt
 
-Basically:
+### Routine
+
+Let's say your language is called `xyz`.
+
 
 - Checkout highlight-js from github...
-- 3rd party languages are placed into the `extra` directory
 
-So if you had a `xyz` language you'd create an `extra/xyz` folder, and that would be your language module. All paths below are relative to that.
+- Create directory `extra/highlightjs-xyz`. This is home directory of your extension. Init repository there, and `package.json`. Here is the structure of your folder.
+
+    ```
+    - dist
+    - - xyz.min.js
+    - src
+    - - xyz.js
+    - test
+    - - detect
```

Comment:
```suggestion
    - src
    - - languages
    - - - xyz.js
    - test
    - - detect
```

Reply:
Please apply the same for the tests... this is preferred and allows multi-language repos very easily.